src/test/resources/application.conf

```diff
@@ -75,26 +94,66 @@ cnp_sanbox {
   httpProxyPort = "8080"
   userCcdId = 6687
   userImportId = 445
-  idamS2SUrl = "http://betaDevBccidamS2SLB.reform.hmcts.net"
+  idamS2SUrl = "http://rpe-service-auth-provider-sandbox.service.core-compute-sandbox.internal"
   idamUserTokenUrl = "http://betaDevBccidamAppLB.reform.hmcts.net"
```

Comment:
This is tactical IDAM.

Can you update this REPO with SIDAM URLs and test it against SIDAM please i.e., SPROD ?

Reply:
done